services/storage/src/simcore_service_storage/dsm.py

```diff
@@ -940,28 +946,22 @@ async def delete_project_simcore_s3(
                 delete_me = delete_me.where(file_meta_data.c.node_id == node_id)
             await conn.execute(delete_me)
 
-        session = aiobotocore.get_session()
-        async with session.create_client(
-            "s3",
-            endpoint_url=self.s3_client.endpoint_url,
-            aws_access_key_id=self.s3_client.access_key,
-            aws_secret_access_key=self.s3_client.secret_key,
-        ) as client:
+        async with self._create_client_context() as client:
```

Comment:
why are new client contests required?

Reply:
I asked myself the same question but this is an  *aiobotocore client* and that is how they use it https://aiobotocore.readthedocs.io/en/latest/#basic-example. I keep a single instance of the session though...